examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,218 @@
+aws iam attach-role-policy \
+--role-name aurora-monitoring \
+--policy-arn arn:aws:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole
+```
+
+Create second IAM role and attach policy to the role (This role is required for aurora proxy to fetch db secrets from secret manager )
+
+```shell
+aws iam create-role \
+--role-name rds-proxy \
+--assume-role-policy-document file://rds-proxy.json
+```
+ 
+ ```shell
+CURRENT_REGION=<REGION_NAME> # replace <REGION_NAME> with current region name e.g us-east-1.
+CLUSTER_NAME=<EKS_CLUSTER_NAME> # replace <EKS_CLUSTER_NAME> with eks cluster name.
+ACCOUNT_NUM=$(aws sts get-caller-identity --query Account --output text)
+```
+
+```shell
+sed -i -e "s/REGION-NAME/$CURRENT_REGION/g" rds-proxy-policy.json
+
+aws iam put-role-policy \
+--role-name rds-proxy \
+--policy-name rds-proxy-policy \
+--policy-document file://rds-proxy-policy.json
+```
+### Provide value to the claim before applying.
+     
+ 1. Open the claim and substitute aurora-monitoring role arn with monitoringRoleArn and rds-proxy role arn with proxyRoleArn respectively.
+ 2. Make sure check the CIDR block for application and provide in the claim. This will used as the security group 
+    ingress rule for proxy from application ( we can also provide the security group id instead of CIDR).    
+    
+### Deploy and verify XRD and composition
+
+Deploy the xrd and composition for aurora
+```shell
+# Assuming root directory.
+cd ./compositions/upbound-aws-provider/aurora
+kubectl apply -k .
+```
+
+```shell
+kubectl get xrds | grep xauroras.db.awsblueprint.io
+```
+
+Expected output:
+
+```shell
+NAME                          ESTABLISHED   OFFERED   AGE
+xauroras.db.awsblueprint.io   True          True      5m
+```
+
+Verify the Compositions
+    
+ ```shell
+kubectl get compositions | grep xauroras.db.awsblueprint.io 
+```
+
+Expected output:
+
+```shell
+NAME                          XR-KIND   XR-APIVERSION                 AGE
+xauroras.db.awsblueprint.io   XAurora   db.awsblueprint.io/v1alpha1   5m
+```
+
+### Apply the Aurora claim
+
+If we have about same kind of output as above we can apply th claim: ( We are using a namespace as team-a in this example, if you want to change , please go ahead and create and update the same in the claim)
+
+ ```shell
+cd ../../../examples/upbound-aws-provider/composite-resources/database-examples/aurora
+kubectl apply -f aurora-postgresql.yaml
+```
+
+We can check the execution of claim by following command:
+
+```shell
+kubectl get Aurora -n team-a
+kubectl describe Aurora -n team-a
+```
+(It should take about 15-20 min to provision the Aurora RDS cluster.)
+Below is the default behaviour of the resource which will be provisioned through the claim, just to mention all this default behaviour can be overridden through patching .
+
+ 1. It will create a 3 node cluster with 1 writer and 2 reader endpoint spread across 3 Azs by default.
+ 2. It will also create a proxy to connect to the aurora cluster.
```

Comment:
There are a lot of mentions about proxies in this doc but not seeing a link or description as to what it is. I think you should at least link to the Aurora doc somewhere.

Reply:
Added the AWS document url.